3rdPartyCypress/cypress/integration/worldServiceCoUkRedirect.js

```diff
@@ -0,0 +1,39 @@
+import services from '../../../src/app/lib/config/services';
+
+Object.keys(services).forEach(index => {
+  const service = index;
```

Comment:
What's the reason for defining this as a constant rather than just making the line above `Object.keys(services).forEach(service => {`